engines/supernova/game-manager.cpp

```diff
@@ -894,6 +912,10 @@ void GameManager::animationOn() {
 }
 
 void GameManager::edit(Common::String &input, int x, int y, uint length) {
+	// turn off keymapper to allow full text input
+	Common::Keymapper *keymapper = g_system->getEventManager()->getKeymapper();
+	keymapper->disableAllGameKeymaps();
+	g_system->setFeatureState(OSystem::kFeatureVirtualKeyboard, true);
```

Comment:
It's great that you though setting the `kFeatureVirtualKeyboard` state here. It's not directly related to the key mapper, but is a good addition. 😄